src/test/regress/citus_tests/query_generator/generate_queries.py

```diff
@@ -1,9 +1,23 @@
+#!/usr/bin/env python3
+
+"""generate_queries
+Usage:
+    generate_queries --seed=<seed>
+
+Options:
+    --seed=<seed>                          Seed number used by the query generator.(ex: 123)
```

Comment:
outdated comment

Reply:
still needed to reproduce previous seed. Just changed seed generation to use current timestamp instead of limited range interval.